rest-api-spec/src/main/resources/rest-api-spec/api/bulk.json

```diff
@@ -74,6 +74,14 @@
       "require_alias": {
         "type": "boolean",
         "description": "Sets require_alias for all incoming documents. Defaults to unset (false)"
+      },
+      "batch_size": {
+        "type": "int",
+        "description": "Sets the batch size"
+      },
+      "batch_ingestion_option": {
+        "type": "string",
+        "description": "A signal to enable batch ingestion"
```

Comment:
According to the RFC there will be other batch ingestion options. The description is incorrect, it's the kind of batch ingestion you want. The options are none, default or parallel. The description here should be "Type of batch ingestion to use" or along those lines.

Reply:
Weird, there's no respond area under your comment on `enum BatchIngestionOption`. 

> I think the allowed options should be "none, default, parallel" and the enums should match.

As explained [in the RFC](https://github.com/opensearch-project/OpenSearch/issues/12457#issuecomment-1978992911), we won't support "parallel" in the near term. So for now, it's only two options: "batch is not enabled" and "batch is enabled". I'm OK with `none`, `default` is pretty vague that user wouldn't know the meaning of it directly. What about "none, batch, parallel_batch"?

Reply:
I like "none" (default), "batch" and "parallel_batch"!

Reply:
"ingestion_option" or just "ingestion" are names with much wider meanings and can put more things like some options only apply to non-batched ingestion, I still prefer `batch_ingestion_option`

Reply:
It feels like we don't need `batch_ingestion_option` is unnecessary until parallel is implemented (which may be much later). Does it make sense to only support `batch_size`, meaning `batch_size: 1` = current behavior, `batch_size: 2+` new batch ingestion behavior?

@reta @andrross what do you think? would you prefer to have an explicit `batch_ingestion_option=none, batch, parallel` or trigger batch ingestion when `batch_size >= 2`?

Reply:
I'm aligned with only using batch_size and it simplifies both user action and implementation.